Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:18
+
+WORKDIR /app
+COPY package*.json ./
+RUN yarn install
+COPY . .
+
+# Lambda web adapter: https://github.com/awslabs/aws-lambda-web-adapter
+# The public.ecr.aws/awsguru/aws-lambda-adapter:0.7.0 image reference in the docs has
+# been pushed to the public CDS ECR to avoid rate limiting when pulling the image.
+COPY --from=public.ecr.aws/cds-snc/aws-lambda-adapter:0.7.0@sha256:00b1441858fb3f4ce3d67882ef6153bacf8ff4bb8bf271750c133667202926af /lambda-adapter /opt/extensions/lambda-adapter
+RUN ln -s /tmp ./.next/cache
+
+EXPOSE 3000
+CMD yarn dev
```

I know very little about Next.js, but could you use the `yarn start` command here?  Unless there's something about the `dev` server that you need for testing.

Also, since this is more of a single-purpose Docker image (only runs your app) it makes sense to use an `ENTRYPOINT` like so:
```suggestion
ENTRYPOINT ["yarn", "start"]
```

Using `CMD` will still work like you expect though.